src/System.Security.Principal.Windows/src/System/Security/Principal/Win32.cs

```diff
@@ -132,7 +132,7 @@ internal static byte[] ConvertIntPtrSidToByteArraySid(IntPtr binaryForm)
 
             try
             {
-                if (TRUE != Interop.mincore.ConvertStringSidToSid(stringSid, out ByteArray))
```

Comment:
@stephentoub I guess that it can reasonably used as a return value, so deleting it is a bad idea.